salt/engines/junos_syslog.py

```diff
@@ -1,44 +1,75 @@
 # -*- coding: utf-8 -*-
 '''
 Junos Syslog Engine
-===================
-
-.. versionadded:: Nitrogen
+==========================
 
 An engine that listen to syslog message from Junos devices,
 extract event information and generate message on SaltStack bus.
 
+One can customize the name of the topic.
+The event data consists of the following fields:
+    a.  hostname
+    b.  hostip
+    c.  daemon
+    d.  event
+    e.  severity
+    f.  priority
+    g.  timestamp
+    h.  message
+    i.  pid
+    j.  raw (the raw event data forwarded from the device)
+The topic can consist of any of the combination of above fields,
+but the topic has to start with ‘jnpr/syslog’.
+So, we can have different combinations like:
+•   jnpr/syslog/hostip/daemon/event
+•   jnpr/syslog/daemon/severity
+The default topic is ‘jnpr/syslog/hostname/event’.
+The topic is to be specified in the configuration file.
+
+The user can choose the type of data he/she wants of the event bus.
+Like, if one wants events pertaining to a particular daemon, he/she can
+specify that in the configuration file:
+    daemon: mgd
+
+One can even have a list of daemons like:
+    daemon:
+      - mgd
+      - sshd
+
 :configuration:
   Example configuration
+
     engines:
       - junos_syslog:
-          port: 516
+          port: 9999
+          topic: jnpr/syslog/hostip/daemon/event
+          daemon:
+            - mgd
+            - sshd
 
 For junos_syslog engine to receive events syslog must be set on junos device.
 This can be done via following configuration:
     set system syslog host <ip-of-the-salt-device> port 516 any any
 
 Here is a sample syslog event which is received from the junos device:
-    ' <30>May 29 05:18:12 bng-ui-vm-92 mspd[1492]: No chassis configuration found '
+  '<30>May 29 05:18:12 bng-ui-vm-9 mspd[1492]: No chassis configuration found'
 
 The source for parsing the syslog messages is taken from:
     https://gist.github.com/leandrosilva/3651640#file-xlog-py
 '''
 
-# Import Python libs
-from __future__ import absolute_import
 import re
 from time import strftime
 import logging
 
-# Import Salt libs
-import salt.utils.event
+import salt
```

Comment:
It's very uncommon to need to do an import at this level. Could this be made more explicit? It appears we only may be using `salt.utils.event` below and if that's all we need, it would be better to stick with that. Please let me know if I missed something here.